{"version": "0.5", "lang": "python", "startTime": "1583660353231", "uuid": "9887fbe6-89db-403b-b53b-1c9e9baa7d9e", "newcontainer": "1", "vmuptime": "1583655513", "platform": "AWS Lambda", "containerID": "2020/03/08/[$LATEST]139ba47634034a848e41f3e88fe46944", "functionName": "ServiceTwoPython", "functionMemory": "3008", "functionRegion": "us-east-1", "vmID": "FMpg2d", "linuxVersion": "#1 SMP Tue Aug 20 23:10:42 UTC 2019", "totalMemory": "3230668", "freeMemory": "3073432", "pageFaults": "70967", "cpuType": "Intel(R) Xeon(R) Processor @ 2.50GHz - Model 62", "cpuModel": "62", "cpuUsr": "54", "cpuNice": "0", "cpuKrn": "109", "cpuIdle": "967720", "cpuIowait": "2", "cpuIrq": "0", "cpuSoftIrq": "1", "vmcpusteal": "29", "contextSwitches": "27826", "frameworkRuntime": "30", "endpoint": "admin13.cluster-cftyxgwyrild.us-east-1.rds.amazonaws.com", "bucketname": "project.bucket.tcss562.sps", "key": "edited_100000_Sales_Records.csv", "error2": "[1041; 'Database was unable to resume within timeout period.']", "userRuntime": "91013", "cpuUsrDelta": "34", "cpuNiceDelta": "0", "cpuKrnDelta": "27", "cpuIdleDelta": "18135", "cpuIowaitDelta": "1", "cpuIrqDelta": "0", "cpuSoftIrqDelta": "5", "vmcpustealDelta": "2", "contextSwitchesDelta": "3209", "pageFaultsDelta": "56477", "frameworkRuntimeDeltas": "16", "runtime": "91059", "endTime": "1583660444290", "2_thread_id": "12", "1_run_id": "0", "zAll": "Final Results:", "roundTripTime": "91829.17", "payload": "{\"bucketname\": \"project.bucket.tcss562.sps\"; \"key\": \"edited_100000_Sales_Records.csv\"; \"tablename\": \"SalesData\"; \"batchSize\": 1000; \"dbEndpoint\": \"admin13.cluster-cftyxgwyrild.us-east-1.rds.amazonaws.com\"; \"dbName\": \"DB_TCSS562\"}", "latency": "770.17", "runtimeOverlap": "20.17"}